source/migration/BuildTasks/Xamarin.AndroidX.Migration.nuspec

```diff
@@ -27,6 +27,9 @@ This package provides a set of tools and MSBuild tasks to aid in the migration f
       <group targetFramework="monoandroid90">
         <dependency id="Xamarin.AndroidX.MultiDex" version="$multidex$" include="All" />
       </group>
+      <group targetFramework="net6.0-android31.0">
```

Comment:
Are we sure this is all that is needed?  It feels like we would need to update all of these lines as well to produce a working package:

https://github.com/xamarin/AndroidX/blob/cc75e785a343352e9d0e0436caa9e59642cea9c8/source/migration/BuildTasks/Xamarin.AndroidX.Migration.nuspec#L38-L52